src/util.ts

```diff
@@ -508,6 +508,10 @@ export function now(): number {
   } else if (typeof process !== 'undefined') {
     const time = process.hrtime();
     return time[0] * 1000 + time[1] / 1000000;
+  } else if (global && global.android !== 'undefined') {
+    return java.lang.System.nanoTime() / 1000000;
+  } else if (global && global.ios !== 'undefined') {
```

Comment:
There is no `global.ios` -- I've tried to get NS team to add one to make this simpler for us plugin developers -- but it doesn't exist; you would have to do something like `if (global && global.NSObject  && global.NSString) {` to determine if you are on iOS.

Reply:
Ah! Let me update that. `global.android` is present, right?